packages/dx-react-scheduler-material-ui/src/templates/drag-drop/resize.jsx

```diff
@@ -0,0 +1,77 @@
+import * as React from 'react';
+import * as PropTypes from 'prop-types';
+import { withStyles } from '@material-ui/core/styles';
+import classNames from 'classnames';
+import {
+  VERTICAL_TYPE, HORIZONTAL_TYPE,
+  POSITION_TOP, POSITION_BOTTOM,
+} from '@devexpress/dx-scheduler-core';
+
+const verticalStyles = {
+  width: '100%',
+  height: '6px',
```

Comment:
Can we use theme.spacing.unit? 8px insted of 6px.

Reply:
Done